modules/administration/pages/tshoot-packages.adoc

```diff
@@ -1,23 +1,40 @@
 [[troubleshooting-packages]]
 = Troubleshooting Package Inconsistencies
 
-
-
-
-
-
-In some cases, updates are available in the web interface, but not appearing on the client.
-If you schedule an update on the client, it will fail with an error stating that no updates are available.
-This can be caused by a metadata regeneration problem, or because update packages have been locked.
-
-The notice that updates are available will appear immediately, but new metadata is only generated on the server after synchronizing.
-In this case, an inconsistency can occur if taskomatic crashes, or because taskomatic is still running and creating new metadata.
-
-To address this issue, check the [path]``/var/log/rhn/rhn_taskomatic_daemon.log`` file to determine if any processess are still running, or an exception which could indicate a crash.
-In the case of a crash, restart taskomatic.
-
-Check package locks and exclude lists to determine if packages are locked or excluded on the client:
-
-On Expanded Support Platform, check [path]``/etc/yum.conf`` and search for ``exclude=``.
-
-On SUSE Linux Enterprise Server, use the [command]``zypper locks`` command.
+////
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
+
+Troubleshooting format:
+
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+////
+
+Packages can sometimes be locked or taskomatic can experience problems, which creates problems with metadata regeneration.
+When this occurs, package updates will be available in the {webui}, but will not appear on the client, and attempts to update the client will fail.
+To correct this, determine if any processes are running, or if a crash could have occured.
+Check package locks and exclude lists to determine if packages are locked or excluded on the client.
+Once you have located the problematic process, the metadata can be regenerated and synchronization occurs as expected.
+
+.Procedure: Resolving Package Inconsisties
+
+. On the client, check the [path]``/var/log/rhn/rhn_taskomatic_daemon.log`` configuratiopn file to determine if any processes are still running, or an exception which could indicate a crash.
```

Comment:
```suggestion
. On the client, check the [path]``/var/log/rhn/rhn_taskomatic_daemon.log`` file to determine if any processes are still running or a crash occured.
```
the log file is not a config file ;) and the rest could be simplified, IMO.

Reply:
And this logfile does not exist on the client, but only on the server.

Reply:
:+1: